frontend/src/Command/Commands/shuffle.ts

```
@@ -13,10 +17,7 @@ class ShuffleCommand extends BaseCommand {
     super("shuffle", "Shuffle", "shuffle", 0, "shuffle", {});
   }
 
-  getSuggestions(
-    _input: string,
-    _parameters: Record<string, string>
-  ): Promise<SuggestionList> {
+  async getSuggestions({}: SuggestionsParams): Promise<SuggestionList> {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)